lsp-ltex.el

```diff
@@ -398,7 +398,10 @@ This file is use to activate the language server."
  (make-lsp-client
   :new-connection (lsp-stdio-connection
                    #'lsp-ltex--server-command
-                   (lambda () (and (file-executable-p (lsp-ltex--server-entry)) (file-exists-p (lsp-ltex--server-entry)))))
+                   (lambda () (let ((server-command (lsp-ltex--server-entry)))
```

Comment:
Maybe change the `server-command` to something more meaningful? Like `entry` or `entry-point`?